recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy >=0.7.0
+    - cython
+    - swig
+
+  run:
+    - python
+    - numpy
+    - scipy >=0.7.0
+
+test:
+  commands:
+    - xraylib
+  imports:
+    - xraylib
+
+about:
+  license: BSD
```

Comment:
This is 3 clause, right? https://github.com/tschoonj/xraylib/blob/master/license_all.txt